app/javascript/tests/public_health/PatientsTable.test.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
```

Comment:
Understanding that this is just a start for the `PatientsTable` frontend tests because there's a lot of functionality here, I was expecting these to include tests for the `localStorage` values at least since that's what this PR is all about. Could they be updated to include assertions on what should/shouldn't be set in local storage?

Reply:
Just to document here, Kylie and I discussed this over a call. Initial investigation into testing local storage had to be time-boxed since there was no simply & obvious path forward identified while researching The need for further tests and investigation into how to test `localStorage` values has been captured in [this Jira ticket](https://tracker.codev.mitre.org/browse/SARAALERT-1271).